client.go

```diff
@@ -176,11 +186,12 @@ func NewNativeClient(user, host, clientVersion string, port int, proxyHost strin
 		ProxyHost:     proxyHost,
 		ProxyPort:     proxyPort,
 		ClientVersion: clientVersion,
+		RemoteKey:     key,
```

Comment:
I don't think it makes sense to have remoteKey a property of the client, while remoteHost a property of the RemoteOutput() command. I think they should either both be a property of the client (passed in through NewNativeConfig), or they should both be a property of RemoteOutput() (passed in via RemoteOutput command). Given the current behavior where a client has a fixed server target, it might make more sense to do the former.